<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>-1</y>
      <width>521</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="openGLLayout">
     <item>
      <widget class="OpenGLWidget" name="openGLWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="meshInfoLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Mesh</string>
       </property>
       <widget class="QCheckBox" name="wireframeCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </widget>
       <widget class="QLabel" name="verticesLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertices :</string>
        </property>
       </widget>
       <widget class="QLabel" name="trianglesLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>66</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Triangles :</string>
        </property>
       </widget>
       <widget class="QLabel" name="verticesCount">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="trianglesCount">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="meshErrorLayout">
     <item>
      <widget class="QLabel" name="meshErrorLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select format :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.obj</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.txt</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeshViewer">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuMeshViewer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>openGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
